File: Serene/Serene.Script/Meeting/MeetingAttendee/MeetingAttendeeDialog.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System.Collections.Generic;

    [IdProperty("MeetingAttendeeId"), IsActiveProperty("IsActive")]
    [FormKey("Meeting.MeetingAttendee"), LocalTextPrefix("Meeting.MeetingAttendee"), Service("Meeting/MeetingAttendee")]
    public class MeetingAttendeeDialog : EntityDialog<MeetingAttendeeRow>
    {
    }
}

File: Serene/Serene.Web/Modules/Meeting/MeetingAttendee/MeetingAttendeeDialog.ts

namespace Serene.Meeting {

    @Serenity.Decorators.registerClass()
    @Serenity.Decorators.responsive()
    export class MeetingAttendeeDialog extends Serenity.EntityDialog<MeetingAttendeeRow, any> {
        protected getFormKey() { return MeetingAttendeeForm.formKey; }
        protected getIdProperty() { return MeetingAttendeeRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingAttendeeRow.localTextPrefix; }
        protected getService() { return MeetingAttendeeService.baseUrl; }

        protected form = new MeetingAttendeeForm(this.idPrefix);

    }
}